src/js/feedback/components/FeedbackForm.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import AlertBox from '../../common/components/AlertBox';
+import ErrorableTextarea from '../../common/components/form-elements/ErrorableTextarea';
+import ErrorableTextInput from '../../common/components/form-elements/ErrorableTextInput';
+
+function FeedbackForm(props) {
+  return (
+    <form
+      className="feedback-form"
+      onSubmit={(event) => {
+        event.preventDefault();
+        props.sendFeedback();
+      }}>
+      {props.errorMessage ? (
+        <div className="feedback-error">
+          <AlertBox status="error" onCloseAlert={props.clearError} content={props.errorMessage} isVisible/>
+        </div>
+      ) : null}
+      <h4 className="feedback-widget-title">Tell us what you think</h4>
+      <div className="va-flex">
+        <div className="feedback-widget-description-container">
+          <ErrorableTextarea
+            label="What can we do to make Vets.gov better?"
+            name="description"
+            errorMessage={props.formErrors.description}
+            field={{ value: props.formValues.description, dirty: false }}
+            onValueChange={({ value: description }) => props.setFormValues({ description })}
+            required/>
+        </div>
+        <div className="feedback-widget-need-help-container">
+          <div className="feedback-widget-need-help-inner">
+            <h3>Need help?</h3>
+            Calls the Vets.gov Help Desk<br/>
+            <a href="tel:18555747286">1-855-574-7286</a><br/>
+            TTY:&nbsp;<a href="tel:+18008778339">1-800-877-8339</a><br/>
+            Monday – Friday, 8:00 a.m. – 8:00 p.m. (<abbr title="eastern time">ET</abbr>)
+          </div>
+        </div>
+      </div>
+      <input
+        id="should-send-response"
+        type="checkbox"
+        value={props.formValues.shouldSendResponse}
+        onChange={({ target: { checked: shouldSendResponse } }) => props.setFormValues({ shouldSendResponse })}/>
+      <label htmlFor="should-send-response">I would like to receive a response about my feedback.</label>
+      <div className="usa-grid-full">
+        <div className="usa-width-one-third">
+          {props.formValues.shouldSendResponse ? (
+            <div className="feedback-email-container">
+              <ErrorableTextInput
+                label="Your email address"
+                name="email"
+                field={{ value: props.formValues.email, dirty: false }}
+                errorMessage={props.formErrors.email}
```

Comment:
Does this show an error as you're typing in an email? One pattern we have on our forms is that we don't show validation errors until you blur a field (to let the user finish typing without being yelled at). After that, validation changes are shown on change.

This PR is making me realize we need an easier way of writing small forms with the best practices we have on our bigger forms.

Reply:
I think the blur is probably a better idea but there is some stuff in place that works okay. I'm gonna clean it up though.